drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -89,7 +98,13 @@ static void nrf5_get_eui64(uint8_t *mac)
 
 #if defined(CONFIG_SOC_NRF5340_CPUAPP) && \
 	defined(CONFIG_TRUSTED_EXECUTION_NONSECURE)
-#error Accessing EUI64 on the non-secure mode is not supported at the moment
+	int ret = spm_request_read(&factoryAddress, (uint32_t)EUI64_ADDR,
+							sizeof(factoryAddress));
+	if (ret != 0) {
+		LOG_ERR("Unable to read EUI64 from the secure zone. \
+				Setting EUI64 to 0");
```

Comment:
Won't this approach put all these tabs in the log message?

Reply:
You're right.